WebContent/index.html

```diff
@@ -61,4 +61,9 @@ <h2 class="sub-heading translate" data-i18n="Getting Started">Getting Started</h
     
     <div id="footer-outer"></div>
   </body>
+  <script src="resources/lib/url/url.min.js"></script>  
+<script src="resources/i18next.min.js" ></script>
+<script src="resources/jquery-i18next.min.js" ></script>
+<script src="resources/i18nextXHRBackend.js"></script>
+<script src="resources/global.js"></script>
```

Comment:
Please move the script references to the top of the HTML page with the rest of the script references (code readibility / maintainability issue)